src/software/new_geom/angle.h

```diff
@@ -444,7 +444,7 @@ inline constexpr double Angle::toRadians() const
 
 inline constexpr double Angle::toDegrees() const
 {
-    return rads / M_PI * 180.0;
+    return rads / M_PI * 180.0f;
```

Comment:
Why was this change needed?

Reply:
Leftover from what I was trying out previously. Removed in latest update